cm/vagrant.yml

```diff
@@ -12,5 +12,22 @@
     - block:
         - apt_key: url=https://www.postgresql.org/media/keys/ACCC4CF8.asc state=present
         - apt_repository: repo="deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" state=present
-        - apt: name=postgresql-9.5 state=present
+        - apt: name={{item}} state=present
+          with_items:
+          - libpq-dev
+          - python-psycopg2
+          - postgresql-9.5
+        - service: name=postgresql state=started
+        - blockinfile:
+            dest: /etc/profile.d/interviews_db.sh
+            create: yes
+            block: |
+              DATABASE_URL=postgresql://vagrant:vagrant@localhost/interviews
```

Comment:
@Shamash2014 heroku defines DATABASES_URL, and we should use it. In this ansible task I export this var for using it in development env.